### PR TITLE
Updated types and stats in configuration file.

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Description
Created configuration file with options for updated types and stats.

## **Discord contact info**
AsparagusEduardo#6051

### CODE REVIEW

**Comment by AsparagusEduardo** [comment]
Waiting until #29 is merged to add updated abilities config.

**Comment by AsparagusEduardo** [comment]
Might as well add updated abilities to the config in a separate PR. Ready for review.

**Review by LOuroboros — APPROVED** [review]
Lgtm.

**Review by ExpoSeed — APPROVED** [review]

**Review by ghoulslash — APPROVED** [review]

**Pull request merged**
